src/inc/gcinfotypes.h

```diff
@@ -376,12 +376,15 @@ enum infoHdrAdjustConstants {
     SET_PROLOGSIZE_MAX = 16,
     SET_EPILOGSIZE_MAX = 10,  // Change to 6
     SET_EPILOGCNT_MAX = 4,
-    SET_UNTRACKED_MAX = 3
+    SET_UNTRACKED_MAX = 3,
+    SET_RET_KIND_MAX = 4,   // 2 bits for ReturnKind
+    SET_BYTES_TO_FOLLOW = 0x80 // If the High-bit of a header or adjustment byte 
```

Comment:
I find the "SET" in the SET_BYTES_TO_FOLLOW name confusing. How about using MORE_BYTES_TO_FOLLOW or HAS_MORE_BYTES instead?